predictor_robojudge.ipynb

```diff
@@ -102,16 +71,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "npis_df = latest_df[[\"CountryName\", \"RegionName\", \"Date\"] + NPI_COLUMNS]"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "npis_df.sample(3)"
+    "# Excpeting 83 regions, plus the empty one '' = 84\n",
+    "len(latest_df.RegionName.unique())"
```

Comment:
Could also make these `assert`s. Even if notebooks don't grind to a halt when asserts fail, it would express in an executable way what the comments affirm.